app/scripts/directives/personTable.js

```diff
@@ -1,36 +1,81 @@
 'use strict';
 
-/**
- * @ngdoc directive
- * @name admissionSystemApp.directive:personTable
- * @description
- * # personTable
- */
-
 angular
 	.module('admissionSystemApp')
   .directive('personTable', personTable);
 
 function personTableController($scope) {
-  var vm = this;
+
+  // filter (checkboxes) options
+	$scope.userFilterPick = [];
+  $scope.oneAtATime = {open : false};  
+  $scope.removeFromUserPick = function (obj) {
+  	var index = $scope.userFilterPick.indexOf(obj);
+  	$scope.userFilterPick.splice(index, 1);
+  };
+
+  // pagination options
+  $scope.maxSize = 5;  
+  $scope.totalItems = 123;
+  $scope.currentPage = 1;
+	
+	// item per page chooser
+	$scope.itemsPerPageOptions = ['10', '25', '50', '100'];
+  $scope.itemsPerPage = $scope.itemsPerPageOptions[1];
+
+  // if page no. changed - do smth
+  $scope.pageChanged = function(pageNumber) {
+    // console.log('page changed to', pageNumber);
+  };
+
+ 	// handle click on the table rows
+  $scope.setSelected = function(item) {
```

Comment:
If this function doesn't do anything else, we can just write: ng-click="selected = item"